<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1191</width>
    <height>732</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1191</width>
    <height>732</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Hệ Thống Quản Lý Nhân Sự</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frameHeader">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1191</width>
      <height>107</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(12, 31, 56);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>286</width>
       <height>107</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>image/logo.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>30</y>
       <width>301</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>28</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white</string>
     </property>
     <property name="text">
      <string>Quản Lý Nhân Sự</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbHello">
     <property name="geometry">
      <rect>
       <x>930</x>
       <y>75</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white</string>
     </property>
     <property name="text">
      <string>Xin chào </string>
     </property>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>108</y>
      <width>1191</width>
      <height>101</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabManage">
     <attribute name="title">
      <string>Quản Lý</string>
     </attribute>
     <widget class="QPushButton" name="btPhongBan">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>0</y>
        <width>75</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>image/icons8-unit-70.png</normaloff>image/icons8-unit-70.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>75</width>
        <height>48</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="btNhanVien">
      <property name="geometry">
       <rect>
        <x>319</x>
        <y>0</y>
        <width>75</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>image/icons8-user-group-70.png</normaloff>image/icons8-user-group-70.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>75</width>
        <height>48</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="btChucVu">
      <property name="geometry">
       <rect>
        <x>577</x>
        <y>0</y>
        <width>75</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>image/baohiem.jpg</normaloff>image/baohiem.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>75</width>
        <height>40</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="btChamCong">
      <property name="geometry">
       <rect>
        <x>448</x>
        <y>0</y>
        <width>75</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>image/icons8-timesheet-70.png</normaloff>image/icons8-timesheet-70.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>75</width>
        <height>48</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="btKhenThuong">
      <property name="geometry">
       <rect>
        <x>706</x>
        <y>0</y>
        <width>75</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>image/icons8-medical-history-70.png</normaloff>image/icons8-medical-history-70.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>45</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="btTinhLuong">
      <property name="geometry">
       <rect>
        <x>835</x>
        <y>0</y>
        <width>75</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>image/icons8-stack-of-money-70.png</normaloff>image/icons8-stack-of-money-70.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>60</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Phòng Ban</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>60</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Nhân Viên</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>60</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Chấm Công</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>60</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Chức Vụ</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>60</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Khen Thưởng</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>60</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Tính Lương</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabSystem">
     <attribute name="title">
      <string>Hệ Thống</string>
     </attribute>
     <widget class="QPushButton" name="btChangePass">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>0</y>
        <width>75</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>image/icons8-password-1-70.png</normaloff>image/icons8-password-1-70.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>75</width>
        <height>48</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="btAddAcc">
      <property name="geometry">
       <rect>
        <x>469</x>
        <y>0</y>
        <width>75</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>image/icons8-add-user-group-man-man-70.png</normaloff>image/icons8-add-user-group-man-man-70.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>75</width>
        <height>48</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>60</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Thêm tài khoản</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>60</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Đổi mật khẩu</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btChangeInfo">
      <property name="geometry">
       <rect>
        <x>598</x>
        <y>0</y>
        <width>75</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>image/changeInforUser.png</normaloff>image/changeInforUser.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>75</width>
        <height>48</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>60</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Thoát</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btExit">
      <property name="geometry">
       <rect>
        <x>727</x>
        <y>0</y>
        <width>75</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>image/icons8-multiply-70.png</normaloff>image/icons8-multiply-70.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>75</width>
        <height>40</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>60</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Đổi thông tin</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabFuncAcces">
     <attribute name="title">
      <string>Chức năng phụ</string>
     </attribute>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>60</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Thống kê chấm công</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btStaticsTimeKeeping">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>0</y>
        <width>75</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>image/icons8-loyalty-card-70.png</normaloff>image/icons8-loyalty-card-70.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>75</width>
        <height>48</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="btSearchNV">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>0</y>
        <width>75</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>image/icons8-user-group-70.png</normaloff>image/icons8-user-group-70.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>75</width>
        <height>48</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>60</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Tìm Kiếm Nhân Viên</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <zorder>btStaticsTimeKeeping</zorder>
     <zorder>label_14</zorder>
     <zorder>btSearchNV</zorder>
     <zorder>label_13</zorder>
    </widget>
   </widget>
   <widget class="QFrame" name="mainFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>208</y>
      <width>1191</width>
      <height>470</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>680</y>
      <width>1191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(12, 31, 56);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <zorder>frameHeader</zorder>
   <zorder>tabWidget</zorder>
   <zorder>frame</zorder>
   <zorder>mainFrame</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
